### PR TITLE
Add Rewrite model

### PR DESCRIPTION
**Pull request by awmackowiak**
Screenshot 2022-11-18 at 08 54 46

### CODE REVIEW

File: vaas/vaas/router/models.py [hunk 1]
```diff
@@ -6,6 +6,25 @@
...
+    ('302', 'Found'),
+    ('307', 'Temporary Redirect')
+)
+
+class Rewrite(models.Model):
+    condition = models.CharField(max_length=512)
+    destination = models.CharField(max_length=512)
+    action = models.CharField(max_length=64, choices=RESPONSE_STATUS_CHICES, default='301')
+    priority = models.PositiveIntegerField()
+    persevre_query_params = models.BooleanField(default=True)
```
Comment (ziollek) [inline vaas/vaas/router/models.py:20]: Pls, check one more time if that spelling is correct

File: vaas/vaas/router/models.py [hunk 2]
```diff
@@ -6,6 +6,25 @@
...
+RESPONSE_STATUS_CHICES = (
+    ('301', 'Move Pernamentlly'),
+    ('302', 'Found'),
+    ('307', 'Temporary Redirect')
+)
+
+class Rewrite(models.Model):
+    condition = models.CharField(max_length=512)
+    destination = models.CharField(max_length=512)
+    action = models.CharField(max_length=64, choices=RESPONSE_STATUS_CHICES, default='301')
```
Comment (ziollek) [inline vaas/vaas/router/models.py:18]: Do we really need 64 characters for that field? Don't you think that int would be enough?

**Review by ziollek — CHANGES_REQUESTED** [review]

File: vaas/vaas/router/models.py (hunk 1 above)
Comment (awmackowiak) [reply vaas/vaas/router/models.py:20]: Preserve will be correct form here. I will change it.

File: vaas/vaas/router/models.py (hunk 2 above)
Comment (awmackowiak) [reply vaas/vaas/router/models.py:18]: I have been think thet this size also is connected whit tuple names but now i see the light. Three are enough.

File: vaas/vaas/router/models.py [hunk 3]
```diff
@@ -6,6 +6,25 @@
...
 from vaas.manager.models import Director
 
+RESPONSE_STATUS_CHICES = (
+    ('301', 'Move Pernamentlly'),
+    ('302', 'Found'),
+    ('307', 'Temporary Redirect')
+)
+
+class Rewrite(models.Model):
+    condition = models.CharField(max_length=512)
```
Comment (smigacz) [inline vaas/vaas/router/models.py:16]: it shouldn't be a field made from three other forms? in our MVP we decide to made condition from method+domain+path
Zrzut ekranu 2022-11-21 o 08 10 38

**Review by smigacz — CHANGES_REQUESTED** [review]

File: vaas/vaas/router/models.py (hunk 3 above)
Comment (awmackowiak) [reply vaas/vaas/router/models.py]: Yes you are right. We will combine condition from this three field exposed in form. 
We have same field in Route:
```python
class Route(models.Model):
    condition = models.CharField(max_length=512)
    ...
```

File: vaas/vaas/router/admin.py [hunk 4]
```diff
@@ -1,11 +1,22 @@
...
 from vaas.external.audit import AuditableModelAdmin
-from vaas.router.models import Route
+from vaas.router.models import Route, Rewrite, RewritePositiveUrl
 from vaas.router.forms import RouteModelForm
 from django.conf import settings
 
 
+class RewritePositiveUrl(admin.TabularInline):
+    model = RewritePositiveUrl
+    extra = 1
```
Comment (ziollek) [inline vaas/vaas/router/admin.py:11]: what is the purpose of this field?

**Review by ziollek — APPROVED** [review]

File: vaas/vaas/router/admin.py (hunk 4 above)
Comment (awmackowiak) [reply vaas/vaas/router/admin.py:11]: Only reason for do that was to exposed it in admin form. To show connection between Rewrites and RewritePositiveUrl. So it is temporarily. Should disapired after form will be add.

**Review by TheMickeyMike — APPROVED** [review]

**Pull request merged**